utils/fuzz/fuzz_targets/prunable.rs

```diff
@@ -0,0 +1,80 @@
+#![no_main]
+
+use arbitrary::{Arbitrary, Unstructured};
+use commonware_utils::bitmap::Prunable;
+use libfuzzer_sys::fuzz_target;
+
+const MAX_OPERATIONS: usize = 10;
```

Comment:
I think this can be larger
```suggestion
const MAX_OPERATIONS: usize = 1024;
```